workbench-core/environments/src/interfaces/listEnvironmentTypeConfigsRequest.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { z } from 'zod';
+
+// eslint-disable-next-line @rushstack/typedef-var
+export const ListEnvironmentTypeConfigsRequestParser = z
+  .object({
+    envTypeId: z.string(),
+    pageSize: z
+      .string()
+      .transform((pageSizeString, ctx) => {
+        const pageSize = parseInt(pageSizeString);
```

Comment:
This looks kind of like the code I wrote. Maybe we abstract this transformer into its own thing so people don't have to copy and past it any time we have a pagination info? Then we could have tests for that specific transformer and wouldn't need to re-test that logic for each individual request.

Reply:
I created a ticket `GALI-2069` for this